src/helper/tools/eye-dropper.js

```diff
@@ -1,11 +1,24 @@
 import paper from '@scratch/paper';
+import {hideGuideLayers, showGuideLayers} from '../layer';
 
 const LOUPE_RADIUS = 20;
+const ZOOM_SCALE = 3;
 
 class EyeDropperTool extends paper.Tool {
     constructor (canvas, width, height, pixelRatio, zoom, offsetX, offsetY) {
         super();
 
+        const guideLayers = hideGuideLayers();
+
+        const colorShot = paper.project.exportSVG({asString: true});
```

Comment:
Your code looks great :-) Are you going to merge this, or do you want me to do something?

I saw the behavior over the button too... No idea how that works LOL, I thought I got rid of it using the blue but somehow thats semi transparent.

Reply:
I have pushed my commit to your branch so you can pull it in.

I think the overlap is caused by the 'Convert to Bitmap' <Button> being on top of the <Loupe> element in scratch-paint/src/components/paint-editor/paint-editor.jsx. It ends up looking see-through because the 'Convert to Bitmap' button has opacity set to .75. It could be fixed by changing the order of the elements so the button is first, and adding some Flexbox CSS to rearrange them, or we could put back 'overflow: hidden' and maybe come back to it later, up to you :)